add release notes up to 10.16.248.0 (#130)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/platform-10-17-0.md

<td>10.16.45.0</td>
</tr>

<tr>
<td>Core platform</td>
<td>On creating a measurement, it is now validated for all processing modes if the measurement exists, that is for PERSISTENT, QUIESCENT, TRANSIENT and CEP.

> Prior to this change, the source existence was only validated for persistence processing modes (PERSISTENT and QUIESCENT).</td>

Reviewer: ```suggestion
Prior to this change, the existence of the measurement was only validated for persistence processing modes (PERSISTENT and QUIESCENT).</td>

```

<td>MTM-50498</td>
<td>10.16.248.0</td>
</tr>

<tr>
<td>Core platform</td>

> <td>Inventory binaries now support chunk download (by setting specific range headers) and resume of the file download.</td>

Reviewer: ```suggestion
<td>Inventory binaries now support chunk download (by setting specific range headers) and resuming a file download.</td>

```
